antlr4ide.tests/src/main/java/log4j.properties

```diff
@@ -12,4 +12,5 @@ log4j.rootLogger=DEBUG, default
 # This appender will write to the stdout console
 log4j.appender.default=org.apache.log4j.ConsoleAppender
 log4j.appender.default.layout=org.apache.log4j.PatternLayout
-log4j.appender.default.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+# log4j.appender.default.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+log4j.appender.default.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} - p - %m%n
```

Comment:
yes, it should be %p (for priority), thanks.

at the moment the console neither uses any hierarchy of loggers nor any customized categories and, therefore, I am pretty sure we can simplify the format this way.